Velaptor/Graphics/Renderers/ITextureRenderer.cs

```diff
@@ -474,4 +663,294 @@ public interface ITextureRenderer
     ///     </list>
     /// </remarks>
     void Render(ITexture texture, Rectangle srcRect, Rectangle destRect, float size, float angle, Color color, RenderEffects effects, int layer = 0);
+
+    /// <summary>
+    /// Renders the given <see cref="Texture"/> using the given parameters.
```

Comment:
Fix all instances of this doc for all render overloads with an atlas texture.

```suggestion
    /// Renders the <see cref="Texture"/> in the given <paramref name="atlasData"/> using the given parameters.
```